installer/conf/container.conf

```diff
@@ -64,3 +64,17 @@
 <filter oms.container.**>
 	type filter_container
 </filter>
+
+#separate buffer for containers
+<match oms.container.**>
+  type out_oms
+  log_level info
+  buffer_chunk_limit 20m
+  buffer_type file
+  buffer_path /var/opt/microsoft/omsagent/state/state/out_oms_docker*.buffer
```

Comment:
Will this conf file ever get onto a VM without a Container? Perhaps that machine's setup is different, but with release 1.3.1-15, I have no ...state/state folder:

`ls: cannot access /var/opt/microsoft/omsagent/state/state/: No such file or directory`